plone/app/layout/permissions.py

```diff
@@ -0,0 +1,10 @@
+# -*- coding: utf-8 -*-
+from AccessControl.SecurityInfo import ModuleSecurityInfo
+from Products.CMFCore.permissions import setDefaultRoles
+
+
+security = ModuleSecurityInfo('plone.app.layout')
+
+security.declarePublic('ShowToolbar')
```

Comment:
maybe is just confused with the various types on permission definitions in CMFPlone, CMFCore, ...